src/docs/development/tools/devtools/memory.md

```diff
@@ -38,77 +47,100 @@ in granularity:
+A timeseries graph is used to visualize the state of the Flutter memory
 at successive intervals of time. Each data point on the chart
 corresponds to the timestamp (x-axis) of measured quantities (y-axis)
 of the heap, for example, usage, capacity, external, garbage
 collection, and resident set size.
 
-![Screenshot of a memory anatomy page]({% asset tools/devtools/memory_anatomy.png @path %}){:width="100%"}
+![Screenshot of a memory anatomy page]({% asset tools/devtools/memory_chart_anatomy.png @path %}){:width="100%"}
+
+The major sections of the memory profiler are:
 
 <dl markdown="1">
-<dt markdown="1">**Legend**</dt>
-<dd>All collected measurements regarding the memory.
-    Clicking a legend name hides or shows that data.</dd>
-<dt markdown="1">**Range selector**</dt>
+<dt markdown="1">**Event Timeline**</dt>
+<dd>Displays DevTools events (such as Snapshot (manual and auto), Dart VM GC, user requested GC, Monitor and Accumulator Reset (button clicks) when they occurred, in relation to the memory chart timeline.</dd>
+<dt markdown="1">**Memory Overview Chart**</dt>
+<dd>This chart is a timeseries graph to help visualize the state of the Dart heap and External Heaps at various points in time.</dd>
+<dt markdown="1">**Android Overview Chart**</dt>
+<dd>This chart, for an Android app, collects Android's ADB tool meminfo section of ADB's App Summary section.</dd>
+<dt markdown="1">**Timeline Position**</dt>
+<dd>Slider controls the x-axis (timeline or left-most position) of the all charts (event, memory and Android).  The increment is based on the current display unit. For example, setting the Display  to 5 minutes then each increment in the slider (from right-most to left) would be now, 5 minutes ago, 10 minutes ago, etc.</dd>
+<dt markdown="1">**Tree View of Memory**</dt>
+<dd>Table Tree view of outstanding memory events e.g., User requested snapshots, Automatic snapshots, Analysis of snapshots, and Memory Allocation monitoring.</dd>
+<dt markdown="1">**Memory View**</dt>
+<dd>Display either the contents of the analysis, snapshot or monitoring based on the currently selected row in the Table Tree view.</dd>
+</dl>
+
+## Memory Controls
+
+The top line of the memory profile page has a number of buttons and dropdowns to control how memory charts are displayed.
+
+![Screenshot of a memory controls]({% asset tools/devtools/memory_controls.png @path %}){:width="100%"}
+
+<dl markdown="1">
+<dt markdown="1">**Pause**</dt>
+<dd>Pause the memory overview chart to allow inspecting
+    the currently plotted data. Incoming memory data is still received;
+    notice the Range selector continues to grow to the right.</dd>
+<dt markdown="1">**Resume**</dt>
+<dd>The memory overview chart is live and displaying the
+    current time and the latest memory data received.</dd>
+<dt markdown="1">**Clear**</dt>
+<dd>All collected data plotted in charts and event timeline are deleted and all snapshots, analyses and allocation monitoring is deleted too.</dd>
```

Comment:
Clear all collected data from the memory profiler

Reply:
Done.